metadata.go

```diff
@@ -80,12 +83,69 @@ var debianLicenseText = map[string]string{
 `,
 }
 
+var githubRegexp = regexp.MustCompile(`github\.com/([^/]+/[^/]+)`)
+
+func findGitHubRepo(gopkg string) (repo string, err error) {
+	if strings.HasPrefix(gopkg, "github.com/") {
+		repo = strings.TrimPrefix(gopkg, "github.com/")
+		return
+	}
+	resp, err := http.Get("https://" + gopkg + "?go-get=1")
+	if err != nil {
+		return
+	}
+	defer resp.Body.Close()
+	z := html.NewTokenizer(resp.Body)
```

Comment:
Are you intentionally not using the Parse method? https://godoc.org/golang.org/x/net/html#example-Parse

It usually results in shorter and easier to follow code :)